.eslintrc.json

```diff
@@ -0,0 +1,42 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+
+    "parser": "babel-eslint",
+
+    "rules": {
```

Comment:
### [OPTIONAL] and a word of caution

Let us practice good judgment when writing rules here. I see that a lot of the rules have been put here to simply pass the linter checks. It isn't a good practice to do that. There's a reason why the style guide was created - to encourage/train us to code professionally. By ignoring a lot of the rules that would have helped us catch code style issues we are essentially preventing any improvements we can make. 

A good rule of thumb in determining what to place in these `rules` is that when a particular linter warning/error you see is not dependent on your coding style but on factors outside your control. For example, `import/no-unresolved` rule at times happens when our system cannot recognize a particular module. If we see this error BUT we know 100% that the module is fully resolved (read by the project file that imports it) then we can safely include a rule "0" for this. However, for things like deconstruction, no-nested-ternary and the like, we should not override them.